includes/admin/class-wc-stripe-rest-payment-intents-controller.php

```
@@ -0,0 +1,103 @@
+						'validate_callback' => 'rest_validate_request_arg',
+					],
+					'ending_before'  => [
+						'type'              => 'string',
+						'required'          => false,
+						'sanitize_callback' => 'sanitize_text_field',
+						'validate_callback' => 'rest_validate_request_arg',
+					],
+				],
+			]
+		);
+	}
```

PHPMD (2.15.0)

[error] The method register_routes is not named in camelCase. (undefined)

(CamelCaseMethodName)